dashboard/views.py

```diff
@@ -2118,6 +2118,31 @@ class DeleteAnswerTranslation(BaseDeleteTranslation):
     model = AnswerTranslation
     template_name = 'dashboard/translations/answer_delete.html'
 
+    def delete(self, request, *args, **kwargs):
+        '''
+        Also delete the question object while we're about it
+        '''
+
+        obj = self.get_object()
+
+        if obj.is_published:
+            question_model = PublishedTranslatedQuestion
```

Comment:
@badrihippo  Its working fine, but here `PublishedTranslatedQuestion` is not imported (nor in our present codebase).

Reply:
Good catch! I've added the line now.